helix-query.yaml

```diff
@@ -70,6 +70,32 @@ indices:
       role:
         select: main h3
         value: innerHTML(el)
+      lastModified:
+        select: none
+        value: parseTimestamp(headers["last-modified"], "ddd, DD MMM YYYY hh:mm:ss GMT")
+  search:
+    include:
+      - '/**'
+    exclude:
+      - '/de/**'
+      - '/de'
```

Comment:
@jvaldas we could remove this to have all languages in one file, but I think to keep the file a little smaller it actually makes sense to separate it and create another index under /de/query-index-search.json later. The thing is you would need to have some logic around this in the lambda, but I assume we would need that logic anyway to separate the two languages into separate indexes. Wdyt?

Reply:
Let’s keep the search solution decoupled from the website. All content extracted and made available, please.
Is there mandatory to use this new target name? I’m lazy. I don’t want to change all the specification files again. 🤣

Reply:
In other words, thanks for the suggestion but please no. 😉

Reply:
@nc-andreashaller, I prefer to have a separate language index under /de/query-index-search.json. DE is out of MVP scope anyway. @AngelaFabregues, I can update the wiki if needed.